cmake/toolchain/xt-clang/Kconfig

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config TOOLCHAIN_XTCLANG_DO_NOT_GENERATE_FLIX
+	bool "Skip instruction fusions via FLIX"
+	depends on DT_HAS_CDNS_TENSILICA_XTENSA_LX7_ENABLED
+	default y if USERSPACE
```

Please turn this setting around so it is disabled per default, and then can be enabled by samples which wants it.
Like:
```suggestion
config TOOLCHAIN_XTCLANG_FLIX
	bool "Flexible Length Instruction Xtensions (FLIX)"
	depends on DT_HAS_CDNS_TENSILICA_XTENSA_LX7_ENABLED
	default y if !USERSPACE
```

Related comment, but not a required change.
Afaik FLIX is just Xtensa's form of `VLIW`, so a more generic option would be to relate the feature to `VLIW` and then let xt-clang define this to be `-mno-generate-flix`.